extensions/ql-vscode/src/common/interface-types.ts

```diff
@@ -577,6 +577,11 @@ interface SetModeledMethodMessage {
   method: ModeledMethod;
 }
 
+interface SetMultipleModeledMethodsMessage {
+  t: "setMultipleModeledMethods";
+  modeledMethods: ModeledMethod[];
```

Comment:
Did we also discuss including `methodSignature: string;` in this message, so you don't have to work out the signature by looking at the modeled methods?

That would also make it easier to remove all modeled methods for a method signature, instead of having to set it to the array containing one element with `type: "none"`.

I'm happy for you to make this change in this PR or I can do it in a followup, because I'll be using this message in my next PR.

Reply:
It does make sense to add it here. Right now, this message is only used for communication from the extension host to the view, but we'll definitely need the signature for the communication from the view to the extension host. I'll change the message now so that I can also use it in a follow-up PR (after #2924).